build.gradle

```
@@ -357,23 +357,37 @@ tasks.register("dockerDistUntar") {
   }
 }
 
-task distDocker(type: Exec) {
+def dockerVariants = [
+  "jdk14",
+  "jdk15"
+]
+
+task distDocker  {
   dependsOn dockerDistUntar
 
   def dockerBuildVersion = rootProject.version
```

With Teku we need to build the docker images as the `develop` version rather than the project version (Besu does it differently because of some historical differences in CI setup and it just never changed back).
```suggestion
  def dockerBuildVersion = 'develop'
```